backend/tests/util.py

```diff
@@ -543,10 +543,6 @@ def setUp(self):
                 self.lora_port,
             )),
             patch('oio_rest.app.settings.LOG_AMQP_SERVER', None),
```

Comment:
Shouldn't this line be removed/replaced as well?

Reply:
> Shouldn't this line be removed/replaced as well?

Good point; although it's a user-facing setting, the _location_ of that variable isn't. (We might move `settings` in the future, as we have in the past.) I'll fix it!